Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,117 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  then
+    git_URL=""
+    git_SHA=""
```

Comment:
Ok, I put git stuff here in order  to illustrate how vendor name choosing will work. Will remove it for this PR.